quic/s2n-quic-core/src/recovery/bandwidth/estimator.rs

```diff
@@ -274,6 +284,13 @@ impl Estimator {
         self.rate_sample.lost_bytes = self.lost_bytes - self.rate_sample.prior_lost_bytes;
     }
 
+    /// Called each time explicit congestion is recorded
+    #[inline]
+    pub fn on_explicit_congestion(&mut self, ce_count: u64) {
+        self.ecn_ce_count += ce_count;
```

Comment:
Isn't this just a normal assignment since ECN counts are monotonic? Or is the caller expected to do a diff first?

Reply:
I convert the raw ECN CE counts into incremental counts in the [ECN controller](https://github.com/aws/s2n-quic/pull/1421/files#diff-010950da0a8e41ef40a6134479ff39342d690d66d876e2ac8ba13a43b11265b3R253).

The CongestionController trait's `on_explicit_congestion` method mentions that it is an incremental count: https://github.com/aws/s2n-quic/pull/1421/files#diff-136a719e5c9014b186e53855bf19a02d8a524f300965fd138041af5fcfcc2089R120